tests/check_templates.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
This script is run after unit tests on TravisCI to check version consistency and test templates.